workflow/common/common.go

```diff
@@ -46,8 +46,9 @@ const (
 	// for the purposes of workflow segregation
 	LabelKeyControllerInstanceID = workflow.WorkflowFullName + "/controller-instanceid"
 	// Who created this workflow.
-	LabelKeyCreator      = workflow.WorkflowFullName + "/creator"
-	LabelKeyCreatorEmail = workflow.WorkflowFullName + "/creator-email"
+	LabelKeyCreator         = workflow.WorkflowFullName + "/creator"
+	LabelKeyCreatorEmail    = workflow.WorkflowFullName + "/creator-email"
+	LabelKeyCreatorUsername = workflow.WorkflowFullName + "/creator-username"
```

Comment:
Should we be honest(?) here? The value is not `username` is is in fact the user preferred username

Reply:
Yep, makes sense. I will make the changes.

Reply:
made the changes, but will require a workflow approval again.